javaharness/README.md

```diff
@@ -39,3 +39,49 @@ and everything that needs to be rebuilt should be.
   ```bash
   bazel clean --expunge
   ```
+
+## Debugging and Inspection
+The Arcs Local Development Server (ALDS) is used to proxy message channels between<br />
```

Comment:
These `<br />` tags don't seem necessary.

Reply:
Thanks! Removed them.

I used to add '\' or '<br />' or two trailing spaces to force line break.
Without any of these magics, some markdown editors cannot present results correctly.
But they are unnecessary in GH :)